assets/css/01-tools/__tools.scss

```diff
@@ -1,6 +1,12 @@
-/**
- * Functions and mixins for project
- */
-
-@import 'functions/_functions';
-@import 'mixins/_mixins';
+/*  =========================================================================
+    tools
+    Default mixins and functions.
+    ========================================================================= */
+
+// Vendor tools
+@import 'node_modules/bourbon/core/bourbon';
+@import 'node_modules/sass-mq/mq';
```

Comment:
@iainvdw basically bourbon and sass-mq are mixin/function libs, so i included them into tools